tembo-operator/src/controller.rs

```diff
@@ -345,11 +345,16 @@ impl CoreDB {
         // Reconcile Pooler resource
         reconcile_pooler(self, ctx.clone()).await?;
 
-        // Check if Postgres is already running
-        let pg_postmaster_start_time = is_not_restarting(self, ctx.clone(), "postgres").await?;
+        // This block of code is currently fairly contentious. What status
+        // elements we should retain in the event the cluster is in a stop
+        // state are currenly unknown and under review. This appears to work
+        // for now, but care should be taken given this status.
 
         let mut new_status = match self.spec.stop {
             false => {
+                // Check if Postgres is already running
+                let pg_postmaster_start_time = is_not_restarting(self, ctx.clone(), "postgres").await?;
```

Comment:
I believe this line is causing the fmt/lint CI job to fail.  Running `cargo fmt` should clear it up.